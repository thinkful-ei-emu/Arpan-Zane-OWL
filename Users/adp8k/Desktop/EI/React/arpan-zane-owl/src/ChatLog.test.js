import React from 'react';
import ReactDOM from 'react-dom';
import ChatLog from './ChatLog'
import renderer from 'react-test-renderer';


describe('<ChatLog />', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');

        ReactDOM.render(<ChatLog />, div);


        ReactDOM.unmountComponentAtNode(div);

    });

    it('renders this UI as expected', () => {
        // Render the component, as JSON
        const tree = renderer.create(<ChatLog />).toJSON();
        // Check whether it matches the previous snapshot
        // Stored in __snapshots__/App.test.js.snap
        expect(tree).toMatchSnapshot(); 
    });
});