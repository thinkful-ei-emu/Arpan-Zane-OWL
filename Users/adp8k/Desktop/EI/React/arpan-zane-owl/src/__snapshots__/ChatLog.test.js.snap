// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChatLog /> renders this UI as expected 1`] = `
<div>
  <div>
    <p>
      message
    </p>
    <p>
      Hello world
    </p>
    <p>
      1548852646559
    </p>
  </div>
  <div>
    <p>
      thumbs-up
    </p>
    <p />
    <p />
  </div>
  <div>
    <p>
      thumbs-down
    </p>
    <p />
    <p />
  </div>
  <div>
    <p>
      raise-hand
    </p>
    <p />
    <p />
  </div>
  <div>
    <p>
      clap
    </p>
    <p />
    <p />
  </div>
  <div>
    <p>
      join
    </p>
    <p />
    <p />
  </div>
  <div>
    <p>
      leave
    </p>
    <p />
    <p />
  </div>
  <div>
    <p>
      join-stage
    </p>
    <p />
    <p />
  </div>
  <div>
    <p>
      leave-stage
    </p>
    <p />
    <p />
  </div>
</div>
`;
